website/src/content/docs/docs/migrate-swagger/faq/mustread.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Understanding the Must-have Breaking Changes
+---
+
+To fully leverage the benefits of TypeSpec and follow best practices, there are some unavoidable breaking changes you will encounter after migrating from Swagger to TypeSpec.
+
+## Using Resources from Common Types
+
+If your resource definition already extends from a resource type in [common-types](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/common-types/resource-management), you can skip this section. Otherwise, you should understand how we identify resource models and select appropriate base models from the TypeSpec resource manager library.
+
+A model is identified as a resource when it meets **all** of the following criteria:
+
+1. There is a GET operation for the model.
+2. At least one operation returns a 200 response containing this model.
+3. The model has properties named `id`, `name`, and `type`.
```

Comment:
'id' , 'name',  and 'type'  have to do with when you can use common types, not whether a type is a resource.